libraries/botbuilder-core/tests/transcriptMiddleware.test.js

```diff
@@ -196,6 +196,54 @@ describe(`TranscriptLoggerMiddleware`, function () {
             adapter.send('test');
         });
     });
+
+    it(`should add resource response id to activity when activity id is empty`, function (done) {
+        var conversationId = null;
```

Comment:
small thing but, should be 'const' in not reassigned, otherwise 'let'

Reply:
fixed